airbyte-integrations/connectors/source-twilio/source_twilio/schemas/conversation_participants.json

```diff
@@ -0,0 +1,86 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Generated schema for Root",
+    "type": "object",
+    "properties": {
+      "participants": {
+        "type": ["null","array"],
+        "items": {
+          "type": "object",
+          "properties": {
+            "account_sid": {
+              "type": "string"
```

Comment:
usually it is used `["null", "string"]` to not break the tests if the field is not required or is empty.